Resources/views/GraphiQL/index.html.twig

```diff
@@ -46,7 +46,7 @@
         var xdebugToken = res.headers.get('X-Debug-Token')
         if (typeof Sfjs !== "undefined" && xdebugToken) {
           var toolbarElement = document.querySelector('.sf-toolbar')
-          var debugUrlPattern = "{{ url('_wdt', {'token': '__TOKEN__'}) }}"
+          var debugUrlPattern = "{{ app.debug ? url('_wdt', {'token': '__TOKEN__'}) : '/_wdt/__TOKEN__' }}"
```

Comment:
Checking that `_wdt` route existing could be the right way to that. [This could maybe help to do that.](http://stackoverflow.com/questions/17241235/test-if-route-exists-in-twig-template-symfony-2)